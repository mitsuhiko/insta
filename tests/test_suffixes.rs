#[test]
fn test_basic_suffixes() {
    for value in vec![1, 2, 3] {
        insta::with_settings!({snapshot_suffix => value.to_string()}, {
            insta::assert_json_snapshot!(&value);
        });
    }
}

#[cfg(feature = "glob")]
#[test]
fn test_basic_globbing() {
    insta::glob!("inputs/*.txt", |path| {
        let contents = std::fs::read_to_string(path).unwrap();
        insta::assert_json_snapshot!(&contents);
    });
}
