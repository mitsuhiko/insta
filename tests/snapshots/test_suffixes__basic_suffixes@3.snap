---
source: tests/test_suffixes.rs
expression: "&value"
---
3
